.gitignore

```diff
@@ -57,3 +57,5 @@ Version.java
 # Vagrant
 .vagrant
 
+# OS file
+.DS_Store
```

Comment:
Could you please remove as it's not project-specific, feel free to add it to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)